pre-lollipop-activity-transition/src/main/java/com/kogitune/activity_transition/core/TransitionAnimation.java

```diff
@@ -73,11 +76,23 @@ private static void runEnterAnimation(MoveData moveData, TimeInterpolator interp
     private static void setImageToView(View toView, String imageFilePath) {
         Bitmap bitmap;
         if (bitmapCache == null || (bitmap = bitmapCache.get()) == null) {
+            synchronized (lock) {
+                while (isReady) {
```

Comment:
Sorry,  I do not understand..
Why loop while isReady?

I think will passed this block.

```
1. saveImage: TransitionAnimation.isReady = false
2. setImageToView synchronized (lock) : not locked, So passed
3. setImageToView while (isReady): isReady is false,So passed
4. setImageToView BitmapFactory.decodeFile: Maybe File not saved yet???
5. saveImage new thread: Image saved
6. saveImage new thread: TransitionAnimation.isReady = true;
7. saveImage new thread: synchronized (TransitionAnimation.lock) 
```